frontend/src/scenes/project-homepage/ProjectHomepage.tsx

```diff
@@ -1,11 +1,106 @@
 import React from 'react'
 import './ProjectHomepage.scss'
 import { PageHeader } from 'lib/components/PageHeader'
+import { Dashboard } from 'scenes/dashboard/Dashboard'
+import { useActions, useValues } from 'kea'
+import { teamLogic } from 'scenes/teamLogic'
+import { SceneExport } from 'scenes/sceneTypes'
+import { DashboardLocation, InsightType } from '~/types'
+import { Button, Row, Typography } from 'antd'
+import { inviteLogic } from 'scenes/organization/Settings/inviteLogic'
+import { navigationLogic } from '~/layout/navigation/navigationLogic'
+import { router } from 'kea-router'
+import { urls } from 'scenes/urls'
+import { LemonSpacer } from 'lib/components/LemonRow'
+import { dashboardLogic } from 'scenes/dashboard/dashboardLogic'
+import { PrimaryDashboardModal } from './PrimaryDashboardModal'
+import { primaryDashboardModalLogic } from './primaryDashboardModalLogic'
+import { HomeIcon } from 'lib/components/icons'
 
 export function ProjectHomepage(): JSX.Element {
+    const { currentTeam } = useValues(teamLogic)
+    const dashboardLogicInstance = dashboardLogic({ id: currentTeam?.primary_dashboard ?? undefined })
+    const { dashboard } = useValues(dashboardLogicInstance)
+    const { closeSitePopover } = useActions(navigationLogic)
+    const { showInviteModal } = useActions(inviteLogic)
+    const { showPrimaryDashboardModal } = useActions(primaryDashboardModalLogic)
+
+    const headerButtons = (
+        <div style={{ display: 'flex' }}>
+            <Button
+                data-attr="project-home-invite-team-members"
+                onClick={() => {
+                    closeSitePopover()
```

Comment:
This seems odd. 1) I can't even click the button when this popover is on top, but 2) the popover itself should handle closing itself when you click outside of it. We can't put this code in every button on every page everywhere :)

Reply:
Ehhh yeah - good call!